README.md

```diff
@@ -4,3 +4,5 @@ This is a temporary repository
 Meine erste Änderung
 
 Meine zweite Änderung
+
+Jkhadkfgjskldjfkl
```

Comment:
Jurij Diaz M... added some commits an hour ago
		Mal gucken, ob's funktioniert 			801d82d
		Merge branch 'master' into jurijs-branch
			1d471f0
@Mulchmann
Member
Mulchmann commented 29 minutes ago

Hm
@Mulchmann
Mulchmann approved these changes 26 minutes ago
README.md
@@ -4,3 +4,5 @@ This is a temporary repository
 Meine erste Änderung
 
 Meine zweite Änderung
+
+Jkhadkfgjskldjfkl
@Mulchmann
Mulchmann 26 minutes ago Member

Hm hm hm
@Mulchmann
Mulchmann 26 minutes ago Member

isich gut